modular_zubbers/code/__DEFINES/bloodsucker_defines.dm

```diff
@@ -172,6 +181,9 @@
 #define COMSIG_SOL_WARNING_GIVEN "comsig_sol_warning_given"
 ///Called on a Bloodsucker's Lifetick.
 #define COMSIG_BLOODSUCKER_ON_LIFETICK "comsig_bloodsucker_on_lifetick"
+/// Called when a Bloodsucker's blood is updated
+#define BLOODSUCKER_UPDATE_BLOOD "bloodsucker_update_blood"
+	#define BLOODSUCKER_UPDATE_BLOOD_DISABLED (1<<0)
```

Comment:
Misplaced tab here.

Reply:
Bitflags used in a signal are usually indented like this

Reply:
and yes, it only has one tab
![image](https://github.com/user-attachments/assets/013d0e4c-7947-469e-9a38-64f47a6e34d5)